cmd/describer.go

```diff
@@ -156,13 +156,21 @@ func (e *ExportsDescriber) Describe() string {
 	table.UTF8Box()
 
 	table.AddTitle("Exports")
-	table.AddHeaders("Name", "Type", "Subject", "Public", "Revocations")
+	table.AddHeaders("Name", "Type", "Subject", "Public", "Revocations", "Monitoring")
```

Comment:
Maybe Tracking, again just trying to be as consistent as we can between programmatic, config, JWT and this tool.